core-bundle/composer.json

```diff
@@ -110,6 +110,7 @@
         "contao/core": "*",
         "contao/manager-bundle": "4.5.* <4.5.2",
         "contao/manager-plugin": "<2.0 || >=3.0",
+        "symfony/mime": "4.4.* <4.4.10 || 5.0.* <5.0.10 || 5.1.0",
```

Comment:
#### Note

In versions prior to `4.4.10`, `5.0.10` and `5.1.1` the automatic detection and parsing of email address strings like

```
Lorem Ipsum <lorem.ipsum@example.org>
```

does not work. That's why I added the conflict in order to make sure this is supported.

Reply:
Contao is not yet compatible with Symfony 5 and most likely we will never support Symfony 5.0 at all (most likely the requirement will be `"symfony/mime": "4.4.* || 5.1.*"`), therefore we should not add the conflict here IMHO.

Reply:
`symfony/mime` in version 5.x can be installed in Contao 4. There is no restriction that prevents that. `symfony/mailer` requires `symfony/mime: ^4.4 | ^5.0`. That's why the conflict needs to extend to `5.*` versions as well, otherwise `symfony/mime` 5.0.0 through 5.0.9 or 5.1.0 might get installed.

Reply:
But the conflict is not that important. Anyone doing a fresh install of Contao 4.10 will likely get the newest version of `symfony/mime` anyway. And anyone updating to Contao 4.10 _should_ get the newest version of `symfony/mime` also.